CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+This Documents will hold all information relevant for contributing to this repository
+
+# ./deploy contribution
```

Comment:
What is it? Is it batyam deployment code?

Reply:
Yes, but in the future it can hold the contribution for the repo as a whole (focused on what was requested for this PR)

Reply:
The remark was in the context of improving the text. Making it clearer. Not a blocker.